### PR TITLE
Subject: add export to BigQuery  Type: new feature

### PR DESCRIPTION
**Pull request by lvvvvvf**

### CODE REVIEW

File: asset/cloud-client/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+public class ExportAssetsBigqueryExample {
+
+  // Use the default project Id.
+  private static final String projectId = ServiceOptions.getDefaultProjectId();
+
+  // Export assets for a project.
+  // @param args path where the results will be exported to.
+  public static void main(String... args) throws Exception {
+    String bigqueryDataset = args[0];
+    String bigqueryTable = args[1];
```
Comment (lesv) [inline asset/cloud-client/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java]: I don't think this matches our guidelines -- especially w/o some kind of message to users when they forget to pass it.

**Comment by lesv** [comment]
Feng - will you need us to merge this?

**Comment by lesv** [comment]
Also, should there be some change to the README for this, like instructions on how to use?

**Comment by lesv** [comment]
@stephaniewang526 @shollyman FYI

**Comment by lesv** [comment]
ping

**Comment by stephaniewang526** [comment]
SGTM!

**Comment by lesv** [comment]
PING.  Do you want this merged?

**Comment by stephaniewang526** [comment]
@shollyman

**Comment by lesv** [comment]
Seems abandoned -- closing the PR

**Pull request closed without merging**